roles/password_policy/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+
+- name: Deactivate any active password policy present in the power HMC
+  ibm.power_hmc.hmc_pwdpolicy:
+      hmc_host: '{{ inventory_hostname }}'
+      hmc_auth:
+          username: "{{ password_policy_hmc_username }}"
+          password: "{{ password_policy_hmc_password }}"
+      state: deactivated
+
+- name: Check the password policies present in the power HMC and store it in a variable
+  ibm.power_hmc.hmc_pwdpolicy:
+      hmc_host: '{{ inventory_hostname }}'
+      hmc_auth:
+          username: "{{ password_policy_hmc_username }}"
+          password: "{{ password_policy_hmc_password }}"
+      state: facts
+      policy_type: policies
+  register: policy_check
+
+- name: Create a new password policy in power HMC if the policy name doesn't already exists
```

Comment:
Correct the name as below
Create a new password policy in power HMC if the policy name doesn't exists.